app/Community/Actions/FormatLegacyCommentPayloadAction.php

```diff
@@ -40,14 +37,19 @@ public function execute(string $payload, bool $isTicketComment): string
 
     private function formatText(string $text): string
     {
+        // First, normalize all existing <br /> tags to \n for consistent processing.
+        $text = preg_replace('/<br\s*\/?>\s*/', "\n", $text);
+
         // Convert special characters to HTML entities.
         $text = htmlspecialchars($text, ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML5);
 
-        // Convert newlines to <br /> tags and normalize multiple line breaks.
-        return preg_replace(
-            '/<br\s*\/?>(\s*<br\s*\/?>)+/',
-            '<br /><br />',
-            nl2br($text)
-        );
+        // Convert newlines to <br /> tags without adding additional \n.
+        $text = str_replace("\n", '<br />', $text);
+
+        // Collapse multiple consecutive line breaks into exactly two.
+        $text = preg_replace('/(<br\s*\/?>\s*){2,}/', '<br /><br />', $text);
```

Comment:
Not sure where this comes into play, but multiple line breaks seem to be preserved:
![image](https://github.com/user-attachments/assets/68580ec0-ff56-4a66-a699-7806b8263b1e)

Reply:
Fixed in latest.